### DISCUSSION TITLE
help! Weights aren’t loading

### DISCUSSION THREAD

**Discussion by jagu000** [flybywiresim/aircraft#5758] [General]
The flight plan loaded into the Aoc but the weights always stay at zero. Nothing I do can change them. Please help. Init page works fine with simbrief. ![image](https://user-images.githubusercontent.com/71265119/131511139-05a76816-6fc0-45e7-9c82-028a8c3ba4bf.jpeg)

**Comment by tracernz** [comment]
Set unit to kg or pounds through the MCDU options (not EFB), or update to latest dev.

**Comment by LH4371** [comment]
just read in another post that there is an issue with the fuel load via MCDU, on my end fuel does not load and the asterix does not blink (whereas it still blinks when loading PAX on the next page)
